src/components/SingleArticle.jsx

```diff
@@ -14,10 +13,20 @@ const SingleArticle = () => {
             console.log(err.response)
         })
     }, [article_id])
+    const ClickHandler = () => {
+        axios.patch(`https://news-williammason.herokuapp.com/api/articles/${article_id}`, {
+            inc_votes: 1
+        }).then((data) => {
+            setArticle(data.data.article)
```

Comment:
At the moment we're sending a patch request and using the response from this to change the vote count to the user which can take some time as the vote count won't change until we hear back from the api. I'd look at the optimistic rendering NCNotes to see how you can amend this so that the user can instantly see the vote increase when they click the button :)

Reply:
On it ty !